src/mailadm/bot.py

```diff
@@ -139,10 +137,10 @@ def is_admin_group_message(self, command: deltachat.Message):
                 if command.get_sender_contact() in command.chat.get_contacts():
                     return True
                 else:
-                    print("%s is not allowed to give commands to mailadm." %
-                          (command.get_sender_contact(),))
+                    logging.log("%s is not allowed to give commands to mailadm.",
```

Comment:
Shouldn't this be logging.info? otherwise, https://docs.python.org/3/library/logging.html#logging.log takes two arguments, the first one being the log level, and not a string;